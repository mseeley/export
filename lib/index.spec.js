/* eslint-env jest */

const path = require("path");
const exportFrom = require("./index");

test("rolling up directory into named exports", () => {
  const actual = exportFrom(path.join(__dirname, "fixture"), {
    verbose: true,
    excludes: ["excludeMe.js"]
  });

  expect(actual).toMatchSnapshot();
});
