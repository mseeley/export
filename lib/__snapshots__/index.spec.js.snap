// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rolling up directory into named exports 1`] = `
Object {
  "DirectoryPascalCase": "DirectoryPascalCase",
  "FilePascalCase": "FilePascalCase.js",
  "directoryCamelCase": "directoryCamelCase",
  "directoryHyphenated": "--directory--hyphenated-",
  "fileCamelCase": "fileCamelCase.js",
  "fileHyphenated": "--file--hyphenated-.js",
}
`;
